### PR TITLE
opencv: update to 3.4.1.

### PR DESCRIPTION
**Pull request by jnbr**

### CODE REVIEW

File: srcpkgs/opencv/template [hunk 1]
```diff
@@ -14,7 +14,7 @@ homepage="http://opencv.org"
 license="3-clause-BSD"
```
Comment (maxice8) [inline srcpkgs/opencv/template]: BSD-3-Clause

Comment (jnbr) [reply srcpkgs/opencv/template]: thanks

**Comment by jnbr** [comment]
Travis runs out of time. Local builds are fine.

**Comment by maxice8** [comment]
On Mon, Apr 23, 2018 at 11:53:33PM +0000, Johannes wrote:
> Travis runs out of time. Local builds are fine.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

add [ci skip] to commits to that travis doesn't even try, it also stops this PR from clogging Travis.

**Comment by jnbr** [comment]
> add [ci skip] to commits to that travis doesn't even try, it also stops this PR from clogging Travis.

Yes, will do that again next time.

**Comment by jnbr** [comment]
rebased

**Comment by jnbr** [comment]
rebased. Local builds are still fine.

**Pull request merged**